### PR TITLE
Added Registration Form Tailwind CSS Component 

### PR DESCRIPTION
**Pull request by ryuk156**
## What type of PR is this? (check all applicable)
PR https://github.com/thedevdojo/tails/issues/3

Hi There. Adding a Registration Form Component. 

- [ ] ✨ Feature

## screenshots
image

## Added to documentation?
- [ ] 🙅 no documentation needed

## [optional] What gif best describes this PR or how it makes you feel?

### CODE REVIEW

**Comment by ryuk156** [comment]
@thinkverse  please review

File: components/Registration-form.html [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+          <input class="appearance-none block w-full bg-gray-200 text-gray-700 border border-gray-200 rounded py-3 px-4 leading-tight focus:outline-none focus:bg-white focus:border-gray-500" id="grid-last-name" type="text" placeholder="Doe">
+        </div>
+      </div>
+      
+      <div class="flex flex-wrap -mx-3 mb-6">
+        <div class="w-full px-3">
+          <label class="block uppercase tracking-wide text-gray-700 text-xs font-bold mb-2" for="grid-email">
+            Email
+          </label>
+          <input class="appearance-none block w-full bg-gray-200 text-gray-700 border border-gray-200 rounded py-3 px-4 mb-3 leading-tight focus:outline-none focus:bg-white focus:border-gray-500" id="grid-email" type="email" placeholder="janedeo@example.com">
```
Comment (thinkverse) [inline components/Registration-form.html]: Placeholder email doesn't match the placeholder's firstname and lastname.
```suggestion
          <input class="appearance-none block w-full bg-gray-200 text-gray-700 border border-gray-200 rounded py-3 px-4 mb-3 leading-tight focus:outline-none focus:bg-white focus:border-gray-500" id="grid-email" type="email" placeholder="janedoe@example.com">
```

File: components/Registration-form.html [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+  
+        </div>
+        <div class="w-full px-3">
+          <label class="block uppercase tracking-wide text-gray-700 text-xs font-bold mb-2" for="grid-number">
+            Number
```
Comment (thinkverse) [inline components/Registration-form.html]: Think that should be `phone number` and not `number`, it's best to be specific. 👍

File: components/Registration-form.html [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+          </label>
+          <input class="appearance-none block w-full bg-gray-200 text-gray-700 border border-gray-200 rounded py-3 px-4 mb-3 leading-tight focus:outline-none focus:bg-white focus:border-gray-500" id="grid-email" type="number" placeholder="+91 XXX XXX XXXX">
```
Comment (thinkverse) [inline components/Registration-form.html]: Best to not use `type[number]` for inputting phone numbers, that will add an increment/decrement style input in most browsers (see screenshot). Stick with `type[tel]` for telephone number, default to regular `text` in unsupported browsers so there won't be an issue with it. 🙂
```suggestion
          <input class="appearance-none block w-full bg-gray-200 text-gray-700 border border-gray-200 rounded py-3 px-4 mb-3 leading-tight focus:outline-none focus:bg-white focus:border-gray-500" id="grid-email" type="tel" placeholder="+91 XXX XXX XXXX">
```

Screen shot showing default increment input

**Review by thinkverse — CHANGES_REQUESTED** [review]
Personally, I would remove the `select` and `checkboxes` sections. These components are plug and play, so those sections won't be used out-of-the-box. 🙂&nbsp; As for the filename, I would rename it to match the component more like `registration-form-full.html` since this seems like a registration form that is a full-fledge one and not one with just _name_, _email_ and _password_. 🤔

I would also work on the spacing a bit more, some groups have their inputs and labels to close so there isn't unified spacing between all elements. 👍

Pssh, you're missing the password input by the way. 🤫

Good first pass though, I would keep working on it some more. Maybe look at [dribbble] or some UI/UX sites for inspiration. 😀

[dribbble]: https://dribbble.com

File: components/Registration-form.html (hunk 3 above)
Comment (ryuk156) [reply components/Registration-form.html]: Will make changes in small time thank you for review

**Comment by ryuk156** [comment]
image

@thinkverse  Done

**Comment by tnylea** [comment]
Thanks, @ryuk156, went ahead and merged this one in, I've made a few UI tweaks and I've renamed it to `form-registration-long.html`.

Thanks for the contribution 🙌

**Comment by ryuk156** [comment]
👍😇

**Pull request merged**